.gitignore

```diff
@@ -1,3 +1,4 @@
 .DS_Store
 old
 .idea
+java/images # generated by fish-pepper
```

Comment:
Should we exclude `tomcat/images` as well?

Reply:
As you can see [here](https://github.com/kubesphere/s2i-java-container/blob/1ddfcad261c4ed3219b1d0d417cc042cfb7fa558/test.sh#L238). Currently, only the dir `java/images` was generated by fish-pepper.

Reply:
Alright.